### PR TITLE
Fix for issue #9652

### LINKED ISSUE #9652: CircleCI Integration tests failing - test_category_section_loads_correct_category
Log Output

```
_____ test_category_section_loads_correct_category[Resolution: 1080x1920] ______
[gw0] linux -- Python 3.6.9 /usr/bin/python3

base_url = 'http://olympia.test'
selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="c98ac172-3349-4339-8871-484bc1c68041")>

    @pytest.mark.desktop_only
    @pytest.mark.nondestructive
    def test_category_section_loads_correct_category(base_url, selenium):
        page = Extensions(selenium, base_url).open()
        item = page.categories.category_list[0]
        name = item.name
        category = item.click()
>       assert name in category.header.name
E       AssertionError: assert 'Alerts & Updates' in 'Searching for add-ons'
E        +  where 'Searching for add-ons' = <pages.desktop.category.Category.Header object at 0x7f7eed7136a0>.name
E        +    where <pages.desktop.category.Category.Header object at 0x7f7eed7136a0> = <pages.desktop.category.Category object at 0x7f7eed713ac8>.header

tests/frontend/ui/test_home.py:42: AssertionError
```

Are there any ideas on what might be causing this? Maybe a category has been renamed or something?

### PR DESCRIPTION
**Pull request by jrbenny35**
Closes #9652

### CODE REVIEW

File: tests/frontend/ui/pages/desktop/category.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ class Header(Region):
 
         _category_name_locator = (By.CLASS_NAME, 'SearchContextCard-header')
 
+        def
```
Comment (bobsilverberg) [inline tests/frontend/ui/pages/desktop/category.py:27]: What is this?

File: tests/frontend/ui/test_home.py [hunk 2]
```diff
@@ -39,6 +42,9 @@ def test_category_section_loads_correct_category(base_url, selenium):
     item = page.categories.category_list[0]
     name = item.name
     category = item.click()
+    WebDriverWait(selenium, 10).until(EC.text_to_be_present_in_element(
```
Comment (bobsilverberg) [inline tests/frontend/ui/test_home.py:45]: This looks good!

**Review by bobsilverberg — APPROVED** [review]
Looks good, thanks for the fix @jrbenny35! Just one question about a statement, that maybe was leftover from other changes?

r+wc

File: tests/frontend/ui/pages/desktop/category.py (hunk 1 above)
Comment (jrbenny35) [reply tests/frontend/ui/pages/desktop/category.py:27]: a thought that I never finished lol

**Comment by willdurand** [comment]
mm it does not look like this patch fixed the issue or do we have a new problem now?

**Pull request merged**